docs/test_plan/proctored_exam.rst

```diff
@@ -0,0 +1,133 @@
+#. Filtering by exam name in the exam selection dropdown should bring up the expected exams (e.g. querying "M4-M6" shows all the exams in that section)
+#. Selecting a specific exam (i.e. the ones you just took) should load the respective exam attempts for those exams
+#. The exam attempts should be in the expected status, e.g. successful attempts are submitted or verified, and the errored attempts are errored
+#. Filtering by username only should show exam attempts for that username
+#. Clicking the "Review Dashboard" link in the modal should load the review dashboard for your chosen Proctoring Software (e.g. Proctortrack or Proctorio)
+
+Can Reject/Verify "Review Required" attempts
+---------------------------------------------------
+INSTRUCTIONS: Go into another exam and be suspiscious. Block your webcam, leave for a few seconds, make strange noises, open a bunch of tabs, watch some youtube, open wikipedia, ask chatgpt for the answer to life, etc. Note you will need to do this a couple of times
+
+#. Submit your suspiscious exam attempt, and check the exams dashboard. Assuming you were suspiscious enough, you should see a row that has a status of "Review Required" AND has a "Review Required" button for your attempt
+    #. NOTE: This exam attempt may appear with the status "Satisfied" if you are using Proctorio. This means Proctorio's system is still analyzing your attempt. If this happens, just wait a bit and refresh the page until the status changes to "Verified"
+#. Clicking the "Review Required" button should spawn a modal that says "Update review status"
+#. Clicking "Cancel" should close the modal
+#. Clicking "Verify" should mark attempt as verified both in the dashboard
+#. Check the Gradebook (Instructor Dashboard -> Student Admin -> Gradebook) and check that the grade for the exam is 100
+#. Clicking "Reject" should mark attempt as reject both in the dashboard
+#. Check the Gradebook (Instructor Dashboard -> Student Admin -> Gradebook) and check that the grade for the exam is 0
+
+Can Review Verified Attempts
+----------------------------
+INSTRUCTIONS: Go into another exam and don't be suspiscious. Keep your webcam on with your face in frame. Complete the exam
+
+#. An exam attempt with a status of "Verified" should appear and have a button/link at the end of their rows titled "Manual Review"
+    #. NOTE: This exam attempt may appear with the status "Satisfied" if you are using Proctorio. This means Proctorio's system is still analyzing your attempt. If this happens, just wait a bit and refresh the page until the status changes to "Verified"
+#. A modal should pop up upon clicking "Manual Review" that contains "Cancel" and "Reject" Buttons
+#. Clicking "Cancel" should close the modal
+#. Clicking "Reject" should change the attempt status to "rejected" in the UI
+#. Upon refreshing a filtering for this exam attempt, it is still marked as "rejected"
```

Comment:
What is this testing? Why don't we do this same step for "Can Reject/Verify "Review Required" attempts"?

Reply:
I'm realizing this test is redundant because the status change in the UI only occurs if the status change in the backend was successful (this is done via "dispatch" in the exam dashboard's thunks), so I'm just going to remove it.